client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
Awesome! Thanks for being so responsive, I really appreciate it! Talk to you soon.